server/controllers/donations.js

```diff
@@ -61,11 +43,11 @@ const stripeDonation = (req, res, next) => {
     return next(new errors.BadRequest('Stripe Token missing.'));
   }
 
-  if (!amountFloat) {
+  if (!amount) {
     return next(new errors.BadRequest('Payment Amount missing.'));
   }
 
-  if (amountInt < 50) {
+  if (amount < 50) {
```

Comment:
By default, all amounts are in cents now. How about the opposite? In cases where it's not (like Paypal expects in dollars), we name it amountNotInCents or amountFloat?